### ISSUE TITLE
Issues with Query Tree in a Form

### ISSUE THREAD

**Issue by aumisb** [cedricvlt/streamlit-condition-tree#2]
Hello, thanks for this great Streamlit component. It has been a great addition and an elegant solution for my app. 

For my use case, I have the query tree builder in a `st.form`. I have noticed that when it loads, the form partially cuts off the top row buttons (whether or not I specify a height). When I reload the page or interact with an UI element, this cutoff disappears. I'm hoping you can reproduce it. I can also  attach a video. 

Here is my minimal example:
```python
import pandas as pd
import streamlit as st

from streamlit_condition_tree import condition_tree, config_from_dataframe

# Initial dataframe
df = pd.DataFrame(
    {
        "First Name": ["Georges", "Alfred"],
        "Age": [45, 98],
        "Favorite Color": ["Green", "Red"],
        "Like Tomatoes": [True, False],
    }
)

# Basic field configuration from dataframe
config = config_from_dataframe(df)

# Condition tree
with st.form("test"):
    query_string = condition_tree(config)
    st.form_submit_button("Submit")
```
 Here is the result:
![SCR-20240219-rzp](https://github.com/cedricvlt/streamlit-condition-tree/assets/46659344/1ed9511d-caa0-49d3-a5be-010f515c9f71)

**Comment by cedricvlt** [comment]
Hi @aumisb,

Setting the min_height parameter should solve your issue.

`query_string = condition_tree(config, min_height=100)`


![image](https://github.com/cedricvlt/streamlit-condition-tree/assets/49170088/a9a93e2f-1be0-4eb7-acf8-12e6bf473031)

**Comment by cedricvlt** [comment]
Hi @aumisb,
Thanks for the issue reporting. I just solved it in the latest version (0.1.3).
